tools/topology/topology2/platform/intel/bt-ssp-config.conf

```diff
@@ -50,6 +56,9 @@ Object.Dai.SSP [
 			tdm_slots	2
 			tx_slots	3
 			rx_slots	0
+			Object.Base.link_config.1 {
+				clock_source	0
```

Comment:
Should this be the cardinal clock (i.e. "1"), so we can use integer divider for 1.536Mhz?

Reply:
no should 0 for xtal as xtal is 38.4 and 38.4M/1.536M = 25

Reply:
@kv2019i I will let you answer this question and follow up with another PR if we need to change the clock source for BT. For now, it actually makes no difference since we dont build the topology for MTL